src/neural/onnx/converter.cc

```diff
@@ -584,18 +596,62 @@ std::string Converter::MakeAttentionBody(OnnxBuilder* builder,
         "/attn_body/reshape2", flow,
         builder->AddInitializer("/const/att_body_shape2",
                                 Int64OnnxConst({-1, 176}, {2})));
+    fist_stage_out_C = 176;
+  } else if (input_embedding == network_format::INPUT_EMBEDDING_PE_DENSE) {
+    int embedding_dense_size = weights.ip_emb_preproc_b.size() / 64;
```

Comment:
This function is huge, maybe move large blocks into separate functions?

Reply:
Done